roles/local-ci_deploy/vars/main/resources.yml

```diff
@@ -1,5 +1,6 @@
 ---
 local_ci_namespace: roles/local-ci_deploy/files/namespace.yml
 local_ci_imagestream: roles/local-ci_deploy/files/imagestream.yml
+local_ci_imagestreamtag: roles/local-ci_deploy/files/operator_helper_image_builder.yml
```

Comment:
name `local_ci_imagestreamtag` isn't used anywhere,
and I think the file `roles/local-ci_deploy/files/operator_helper_image_builder.yml` can be removed as I don't see it being used

Reply:
ok